django/core/serializers/python.py

```diff
@@ -91,6 +92,7 @@ def Deserializer(object_list, **options):
     """
     db = options.pop('using', DEFAULT_DB_ALIAS)
     ignore = options.pop('ignorenonexistent', False)
+    handle_forward_references = options.pop('handle_forward_references', False)
```

Comment:
I put this in for backwards compatibility, but it's probably not necessary -- I don't see why anybody would be relying on the current behaviour of being unable to load forward references.